### PR TITLE
Miceoverse : Ask My coding partner for a review 5/9/2022.

### PR DESCRIPTION
**Pull request by khitermedachraf**
## Kindly you can use these Instructions as hints in your process:

- Once you get a pull request link from your partner, go ahead and review it:
    - Reminder: [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).
- Check the quality of the code (you can use previous lessons as a guide). Remember to check for the following:
    - [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md ).
    - [DRY, KISS, and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) rules.
    - [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Finish your review with a nice summary.

### CODE REVIEW

**Comment by nedjwab** [comment]
### Hi @khitermedachraf ,

Well Done implementing the projects requirements👏 
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!



## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <title>My To-Do list</title>
+  </head>
+  <body>
+    <header><h1>My To Do list:</h1></header>
+    <main class="container">
+      <div class="list-container">
+        <ul id="todo-list">
+          <li>
+            <p>Today's To DO</p>
+            <svg
```
Comment (nedjwab) [inline src/index.html:15]: - [x] you can add a javaScript function for the refresh button so when the user clicks here the page loads.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    userTask.value = '';
+    const currentId = e.target.parentElement.parentElement.parentElement.id;
+    Actions.removeTask(currentId);
+    Actions.displayTasks(todoTasks);
+  } else if (e.target.classList.contains('description')) {
+    // Implement the functionality for editing a task
+    userTask.focus();
+    // Iterate throw list items to setback the initial background color
+    const allLi = todoContainer.childNodes;
+    for (let i = 0; i < allLi.length; i++) {
```
Comment (nedjwab) [inline src/index.js]: - [x] kindlly use foreach instead of using for loops , it's one of the javaScript best practices

**Review by nedjwab — COMMENTED** [review]
###  you can add a javaScript function to this refresh icon ,so when the user clicks on it the page reloads.

File: src/index.js (hunk 2 above)
Comment (khitermedachraf) [reply src/index.js]: **Done ✔**

**Comment by khitermedachraf** [comment]
and Thank you so much Zin😂 , for your re-review.

**Comment by nedjwab** [comment]
Hi @khitermedachraf , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**